src/zoid/buttons/util.js

```diff
@@ -106,6 +108,21 @@ export function getVenmoExperiment(experiment : ?Experiment) : VenmoExperiment {
     };
 }
 
+export function getRenderedButtons(props : ButtonProps) : $ReadOnlyArray<$Values<typeof FUNDING>> {
+    const { fundingSource, onShippingChange, style = {}, fundingEligibility = getRefinedFundingEligibility(),
+        experiment = getVenmoExperiment(createVenmoExperiment()), applePaySupport, supportsPopups = userAgentSupportsPopups(),
```

Comment:
Currently calling `createVenmoExperiment()` is going to create a new belter client-side experiment each time. We currently call it here: https://github.com/paypal/paypal-checkout-components/blob/master/src/zoid/buttons/component.jsx#L27 and pass that value down to `getVenmoExperiment()`. 

I think the easiest solution is to memoize the `createVenmoExperiment()` function. That way we don't have to worry about if it gets called multiple times. Can you update the `createVenmoExperiment()` function to use memoize from belter?

Reply:
Here's what I'm thinking for adding memoize to createVenmoExperiment():

```js
const createVenmoExperiment : Experiment | void = memoize(() => {
    const enableFunding = getEnableFunding();
    const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1;

    const fundingEligibility = getFundingEligibility();
    const isEligibleForVenmo = fundingEligibility && fundingEligibility[FUNDING.VENMO] && fundingEligibility[FUNDING.VENMO].eligible;

    // exclude buyers who are not eligible
    // exclude non-desktop integrations using enable-funding=venmo
    if (!isEligibleForVenmo || (isEnableFundingVenmo && isSupportedNativeBrowser())) {
        return;
    }

    if (isIos() && isSafari()) {
        return createExperiment('enable_venmo_ios', 90);
    }

    if (isAndroid() && isChrome()) {
        return createExperiment('enable_venmo_android', 90);
    }

    if (!isDevice()) {
        return createExperiment('enable_venmo_desktop', 0);
    }
});
```

Reply:
I just tested it and all this information is coming in the props correctly. It's much simpler than I did

Reply:
Could you please review it again?

Reply:
@lucasfrotam I'm not sure I understand. Are you saying that the venmo information is already passed in the props so you don't have to call the venmo functions at all?

Reply:
I talked to Greg and I really need to call the functions I was calling. I'm adding the functions again and I'm going to add the `memoize`

Reply:
Done!